backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/GraphQLContext.kt

```diff
@@ -16,10 +16,15 @@ data class GraphQLContext(
     val backendConfiguration: BackendConfiguration,
     val regionIdentifierByPostalCode: List<Pair<String, String>>,
     val request: HttpServletRequest,
-) : GraphQLContext {
-    fun enforceSignedIn(): JwtPayload {
-        val isSignedIn = jwtPayload != null
-        if (!isSignedIn) throw UnauthorizedException()
-        return jwtPayload!!
-    }
+) {
+    fun enforceSignedIn(): JwtPayload = jwtPayload ?: throw UnauthorizedException()
 }
+
+private const val ID = "context"
+
+fun GraphQLContext.Builder.put(context: app.ehrenamtskarte.backend.common.webservice.GraphQLContext) {
+    put(ID, context)
+}
+
+val GraphQLContext.context: app.ehrenamtskarte.backend.common.webservice.GraphQLContext
+    get() = get(ID)
```

Comment:
```suggestion
fun GraphQLContext.Builder.put(context: app.ehrenamtskarte.backend.common.webservice.GraphQLContext) {
    put(GraphQLContext::class, context)
}

val GraphQLContext.context: app.ehrenamtskarte.backend.common.webservice.GraphQLContext
    get() = get(GraphQLContext::class)
```

:upside_down_face: type can be used as a key